Formula/zsh-vi-mode.rb

```diff
@@ -14,13 +15,16 @@ def install
 
   def caveats
     <<~EOS
-      To activate the zsh vi mode, add the following at the end of your .zshrc:
+      To activate the zsh vi mode, add the following line to your .zshrc:
         source #{opt_pkgshare}/zsh-vi-mode.plugin.zsh
+
+      To check the plugin version, use the following command after activation:
```

Comment:
This should be in the documentation, it's not homebrew specific.

Reply:
Ok, just modified the caveats.